app/controllers/gardens_controller.rb

```diff
@@ -1,6 +1,6 @@
 class GardensController < ApplicationController
   before_action :authenticate_member!, except: %i(index show)
-  after_action :expire_homepage, only: %i(create delete)
+  after_action :expire_homepage, only: %i(create destroy)
```

Comment:
this is the cause of the bug - where plantings from deleted gardens appear on the homepage